exchanges/okgroup/okgroup_wrapper.go

```diff
@@ -262,7 +263,7 @@ func (o *OKGroup) GetFundingHistory() (resp []exchange.FundHistory, err error) {
 }
 
 // GetExchangeHistory returns historic trade data since exchange opening.
-func (o *OKGroup) GetExchangeHistory(p currency.Pair, assetType asset.Item) ([]exchange.TradeHistory, error) {
+func (o *OKGroup) GetExchangeHistory(*trade.HistoryRequest) ([]trade.History, error) {
 	return nil, common.ErrNotYetImplemented
```

Comment:
Out of scope. But should probably set up a trello card to implement this and any other pending exchange items relating to trade history.